docs/src/pages/components/text-fields/BasicTextFields.js

```diff
@@ -1,24 +1,20 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    '& > *': {
-      margin: theme.spacing(1),
-      width: '25ch',
-    },
-  },
-}));
-
 export default function BasicTextFields() {
-  const classes = useStyles();
-
   return (
-    <form className={classes.root} noValidate autoComplete="off">
+    <Box
+      component="form"
+      sx={{
+        '& > :not(style)': { m: 1, width: '25ch' },
```

Comment:
A bit late to the party, so I got a question :) Why do we need the `:not(style)` selector?

Reply:
1. To increase the specificity 
Not that much relevant: 2. To not target the emotion style attributes (if a developer copy and paste the full demo)